cmd/notary/config.json

```diff
@@ -0,0 +1,7 @@
+{
+	"remote_server": {
+		"url": "https://localhost:4443",
```

Comment:
The hardcoded default is `https://notary-server:4443` so it would make sense to use that here too.